Add initial Online conference prospectus. (#1235)
writethedocs/www: docs/about/events-activities.rst

We have meetups in multiple cities every month. To find a meetup near you, take a look at the :doc:`/meetups/index` page.

If you want to start a new meetup, our :doc:`/organizer-guide/meetups/starting` page has the information that you need.

Sponsorship
~~~~~~~~~~~

Find ways to connect with our community on the :doc:`/sponsorship/index` pages of our site.

> We have a number of diffenent options and would love your help in supporting our community.

Reviewer: ```suggestion
We have multiple sponsorship levels with different perks, and we would love your help in supporting our community.
```

> We have multiple sponsorship levels with different perks, and we would love your help in supporting our community.

Reviewer 2: ```suggestion
Pick a sponsorship level and perks that suit you. We would love your help in supporting our community.
```
